apps/next/package.json

```diff
@@ -63,6 +64,7 @@
     "postgres": "^3.4.4",
     "react": "^18",
     "react-dom": "^18",
+    "sharp": "^0.33.3",
```

Comment:
what's this for?

Reply:
It was recommended by Next.js for building images for production. Details are here:
https://nextjs.org/docs/messages/install-sharp